commandline/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,43 @@
+package org.example;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class Main {
+    public static List<User> readUsersFromJson(String filename) throws IOException {
+        FileIO fileIO = new FileIO(filename);
+        String jsonContent = Files.readString(fileIO.filePath);
+        ObjectMapper objectMapper = new ObjectMapper();
+        List<User> users = objectMapper.readValue(jsonContent, new TypeReference<List<User>>() {});
```

Comment:
could you return this instead of making a new variable? I don't actually know but have to leave some kind of comment in a pretty perfect code :)

Reply:
done, thank you!